packages/libs/sdk-mixins/src/mixins/descopeUiMixin/descopeUiMixin.ts

```diff
@@ -75,6 +77,12 @@ export const descopeUiMixin = createSingletonMixin(
               `Cannot load DescopeUI from fallback URL, Make sure this URL is valid and return the correct script: "${fallbackScriptEle.src}"`,
             ),
           );
+
+          // in case we could not load DescopeUI from the main URL, we are trying to load it from a fallback URL
+          this.#handleFallback2Script(
+            fallbackScriptEle[this.#errorCbsSym],
+            fallbackScriptEle[this.#loadCbsSym],
+          );
         });
```

Comment:
Instead of duplicating this code, we can reuse the `#handleFallbackScript` function 
We can add more parameters to it if needed

Let me know if you’d like to work on it together